api/config.go

```diff
@@ -143,6 +144,12 @@ func LoadDefaultConfig(args []string) (config Config, err error) {
 
 	config.setFlags(flagSet)
 
+	// read env variable DDT_EXTRA_ARGS and append the arguments
+	envArgs := os.Getenv("DDT_EXTRA_ARGS")
```

Comment:
please don't do this. systemd will take care of adding DDT_EXTRA_ARGS to the command line.